From: Christoph Hellwig <hch@lst.de>
Subject: raid6: enable lock context analysis
Date: Wed, 15 Jul 2026 16:47:36 +0200

The code doesn't have any locking, so this is trivial.

Link: https://lore.kernel.org/20260715144825.95432-6-hch@lst.de
Signed-off-by: Christoph Hellwig <hch@lst.de>
Cc: Eric Biggers <ebiggers@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/raid/raid6/Makefile |    2 ++
 1 file changed, 2 insertions(+)

--- a/lib/raid/raid6/Makefile~raid6-enable-lock-context-analysis
+++ a/lib/raid/raid6/Makefile
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 
+CONTEXT_ANALYSIS := y
+
 ccflags-y			+= -I $(src)
 
 ifeq ($(CONFIG_RAID6_PQ_ARCH),y)
_
